test/json/createInvoiceRequest.json

```diff
@@ -18,7 +18,6 @@
   },
   "currency": "USD",
   "extendedNotifications": true,
-  "forcedBuyerSelectedTransactionCurrency": null,
```

Comment:
@mwarzybok-sumoheavy I noticed this is being removed and there was a typo in the dev docs, but I've updated that -- `forcedBuyerSelectedTransactionCurrency` is a valid request param.

Reply:
it's inside test, not model, same like below. I wasn't sure about this parameter so I removed it ONLY from tests